pegasus/sites.v3/code.org/public/ai.haml

```diff
@@ -16,13 +16,18 @@ theme: responsive_full_width
 - icon_book = "fa-solid fa-book"
 - icon_person_chalkboard = "fa-solid fa-person-chalkboard"
 - icon_video = "fa-solid fa-video"
+- icon_megaphone = "fa-solid fa-megaphone"
 
 %section.hero-banner-basic.bg-neutral-dark{style: "display: block !important"}
   .wrapper
     .text-wrapper.col-60
       %h1{style: "color: white; margin-bottom: 0"}=hoc_s(:ai_hero_heading_01)
       %h2.heading-xxl{style: "color: white"}=hoc_s(:ai_hero_heading_02)
       %p{style: "color: white"}=hoc_s(:ai_hero_desc)
+      - if !!DCDO.get('teach-ai-launch-2023', false)
+        %a.link-button{href: "#teach-ai-callout"}
+          %i{class: "#{icon_megaphone}", style: "margin-right: 6px"}
```

Comment:
I need to make a button class that will work with icons for rtl languages bit need to think about it more and will do that in another task.

Reply:
Sounds good!